src/ssh/mock-sshd.c

```diff
@@ -575,6 +575,12 @@ mock_ssh_server (const gchar *server_addr,
   ssh_bind_options_set (sshbind, SSH_BIND_OPTIONS_RSAKEY, SRCDIR "/src/ssh/mock_rsa_key");
   ssh_bind_options_set (sshbind, SSH_BIND_OPTIONS_DSAKEY, SRCDIR "/src/ssh/mock_dsa_key");
 
+  /* Known issue with recent libssh versions on 32bits: avoid using
+   * curve25519-sha256.
```

Comment:
Can you please add the URL to the upstream bug here, so that it's easier to find in the future?